.github/workflows/wheels-manylinux-publish.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish RAPIDS manylinux wheels
+
+on:
+  workflow_call:
+    inputs:
+      # repo and branch
+      repo:
+        type: string
+      branch:
+        type: string
+      date:
+        type: string
+      sha:
+        type: string
+      build_type:
+        required: true
+        type: string
+
+      # general settings
+      package-name:
+        required: true
+        type: string
+
+      # twine settings
+      build-tag:
+        required: false
+        type: string
+        default: ''
+
+jobs:
+  wheel-publish:
+    name: wheels publish
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.8', '3.9']
+        linux-arch: ["x86_64", "aarch64"]
+        ctk: ['11.5.1']
+    container:
+      # ctk version of the cibw container is irrelevant in the publish step
+      # it's simply a launcher for twine
+      image: "rapidsai/cibuildwheel:cuda-runtime-11.8.0-ubuntu20.04"
+      env:
+        RAPIDS_PY_WHEEL_BUILD_TAG: ${{ inputs.build-tag }}
+        RAPIDS_BUILD_TYPE: ${{ inputs.build_type }}
+        RAPIDS_ARCH: ${{ matrix.linux-arch }}
+        RAPIDS_PY_VERSION: ${{ matrix.python }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.RAPIDSAI_GHA_AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.RAPIDSAI_GHA_AWS_SECRET_ACCESS_KEY }}
+        TWINE_USERNAME: cibuildwheel
```

Comment:
Out of scope for now, but I would probably rename our user on k8s to rapids or something rather than cibuildwheel.